packages/sw-appcache-behavior/build.js

```diff
@@ -32,7 +32,7 @@ module.exports = () => {
           commonjs(),
         ],
       },
-      outputName: 'build/client-runtime.js',
+      buildPath: 'build/client-runtime.js',
```

Comment:
Ignore if that's the case. Just wanted to make sure that was the case.

I'm going to hit the same problem with some push modules I intend to write next quarter (i.e. lib in window and lib in SW). Just leave as is for now.